src/realm/realm.js

```diff
@@ -73,8 +73,24 @@ const authSchema = {
 const realm = new Realm({
   path: 'esteem.realm',
   schema: [userSchema, authSchema, draftSchema, settingsSchema, applicationSchema, scAccounts],
+  schemaVersion: 0,
```

Comment:
Adding schema and migration function won't cause issue for first alpha version?

Reply:
I tried with new scheme, and also I tried different scenario, it is working good

Reply:
nice, let's merge it then